examples/test_code.c

```diff
@@ -5,17 +5,7 @@
 #include "gps/gps.h"
 #include <string.h>
 #include <stdio.h>
-#include <math.h>
-
-#define RUN_TEST(x)         do {                \
-    if ((x)) {                                  \
-        printf("Test passed on line %u with condition " # x "\r\n", (unsigned)__LINE__);\
-    } else {                                    \
-        printf("Test FAILED on line %u with condition " # x "\r\n", (unsigned)__LINE__ );   \
-    }                                           \
-} while (0)
-#define FLT_IS_EQUAL(x, y)      (fabs((double)(x) - (double)(y)) < 0.00001)
-#define INT_IS_EQUAL(x, y)      ((int)((x) == (y)))
+#include <test_common.h>
```

Comment:
Non C standard includes shall have quotes for include, not `<>` characters.

Reply:
ok, see 3d27b2f3